### PR TITLE
The text on the Become... button corresponds to the content of the block

### LINKED ISSUE #93: [SignUp] FrontEnd-01 #93 The text on the button 'Become' (How It Works block) doesn't correspond the requirements
Defect Type: UI/Usability
Severity: Trivial
Priority: Low
Reproducible: Always
Environment: Windows 10 21H2, Chrome 110.0.5481.178

Description:
The text on the button "Become a ..." doesn't match the content (How-it-Works block).

Preconditions:
The web-page on the client is opened, button "How it Works" is clicked. We're now in "How it Works" block.

Steps to reproduce:
Take a look at the button below.

Actual result:
The text on the button "Become a ..." doesn't match the content.

Image

Expected result:
The text on the button must correspond to the selected role.

### PR DESCRIPTION
**Pull request by OlhaOvsiienko**

### CODE REVIEW

File: src/containers/guest-home-page/how-it-works/HowItWorks.jsx [hunk 1]
```diff
@@ -49,7 +49,7 @@ const HowItWorks = () => {
         <CardsWithButton
           array={isStudent ? tutorCardBoxArray : studentCardBoxArray}
```
Comment (khytrykv) [inline src/containers/guest-home-page/how-it-works/HowItWorks.jsx]: I think that `isStudent` is calculated wrongly. We should check it or just rename it.

**Review by khytrykv — APPROVED** [review]

**Review by holus-bolus — APPROVED** [review]

**Review by YaroslavChuiko — APPROVED** [review]

File: src/containers/guest-home-page/how-it-works/HowItWorks.jsx [hunk 2]
```diff
@@ -40,21 +40,21 @@ const HowItWorks = () => {
...
+          array={isTutor ? tutorCardBoxArray : studentCardBoxArray}
           btnText={
-            isStudent
+            !isTutor
               ? t('guestHomePage.whatCanYouDo.learn.actionLabel')
               : t('guestHomePage.whatCanYouDo.teach.actionLabel')
           }
-          isStudent={isStudent}
-          role={isStudent ? tutor : student}
+          isStudent={isTutor}
```
Comment (vlad-khrychov) [inline src/containers/guest-home-page/how-it-works/HowItWorks.jsx]: This is a strange naming, why are we using properties called `isStudent` with a value of `isTutor`?

**Review by vlad-khrychov — CHANGES_REQUESTED** [review]

File: src/containers/guest-home-page/how-it-works/HowItWorks.jsx (hunk 2 above)
Comment (khytrykv) [reply src/containers/guest-home-page/how-it-works/HowItWorks.jsx]: The naming was wrong from the very beginning. We fixed it in the `HowItWorks` component. Should we fix it in the `CardsWithButton` component in the scope of this PR?

Comment (OlhaOvsiienko) [reply src/containers/guest-home-page/how-it-works/HowItWorks.jsx]: In 'How It Works', the property of the CardWithButton component has been corrected.

**Pull request merged**